### PR TITLE
Add Icons for LSP_KINDS

### PR DESCRIPTION
**Pull request by CKolkey**
I thought it would be nice to add some icons for different LSP Kinds.

Screen Shot 2021-02-05 at 11 44 41

### CODE REVIEW

**Comment by jubnzv** [comment]
Will this require having specific fonts with icon support? Or external plugins like vim-devicons?
I believe that the user should be able to disable this option, since not everyone uses such fonts.

**Comment by CKolkey** [comment]
This would require a font with icon support, so I think your idea is sensible. I'll put it behind an option.

**Comment by CKolkey** [comment]
@jubnzv I've added some documentation, a default value for the variable (I figured this should be opt-in). I don't usually write python, so let me know if this could be more idiomatic or cleaner.

**Comment by Shougo** [comment]
Please add the patched font example in the documentation.

File: rplugin/python3/deoplete/source/lsp.py [hunk 1]
```diff
@@ -47,6 +75,7 @@ def __init__(self, vim):
         self.rank = 500
         self.input_pattern = r'[^\w\s]$'
         self.is_volatile = True
+        self.labels = LSP_KINDS_WITH_ICONS if vim.eval('g:deoplete#lsp#use_icons_for_candidates') else LSP_KINDS
```
Comment (Shougo) [inline rplugin/python3/deoplete/source/lsp.py:78]: This line is too long.

Comment (CKolkey) [reply rplugin/python3/deoplete/source/lsp.py]: 👍🏼

**Comment by Shougo** [comment]
Thanks.

**Comment by CKolkey** [comment]
Happy to help. Keep up the great work :)

**Comment by Shougo** [comment]
I have improved the implementation.
Please check it.

**Pull request merged**